netcam_rtsp.c

```diff
@@ -258,15 +258,31 @@ struct rtsp_context *rtsp_new_context(void){
 static int netcam_interrupt_rtsp(void *ctx){
     struct rtsp_context *rtsp = (struct rtsp_context *)ctx;
 
-    if (rtsp->readingframe != 1) {
+    if (rtsp->status == RTSP_CONNECTED) {
```

Comment:
I'm not sure I understand the reasoning behind not having a timeout in the RTSP_CONNECTED status.